src/Features/Core/Portable/ChangeSignature/ChangeSignatureTelemetryLogger.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.CodeAnalysis
```

Comment:
nit: Should this be in our root namespace?

Reply:
Some of the other ones are for some reason. I've moved this one though.